python/lsst/ap/verify/ap_verify.py

```diff
@@ -153,11 +153,19 @@ def _measure_final_properties(metadata, output_dir, args, metrics_job):
         metrics_job.measurements.insert(measurement)
 
 
-def run_ap_verify():
+def run_ap_verify(cmdLine=None):
```

Comment:
is there a standard way to do this in LSST?  It seems unconventional...

Reply:
There's [CmdLineTask.parseAndRun](https://lsst-web.ncsa.illinois.edu/doxygen/x_masterDoxyDoc/classlsst_1_1pipe_1_1base_1_1cmd_line_task_1_1_cmd_line_task.html#a5820e5ed05ca70b8e58efa8ee0567cdf)...

Reply:
Okay, part of the DM-11372 type work then.